### PR TITLE
[merged] Add substitutions for Opt variables

### PR DESCRIPTION
**Pull request by rhatdan**
I think this will fix https://github.com/projectatomic/atomic/issues/540

### CODE REVIEW

**Comment by rhatdan** [comment]
@adelton PTAL

**Comment by rhatdan** [comment]
@adelton PTAL

File: Atomic/atomic.py [hunk 1]
```diff
@@ -304,13 +304,13 @@ def cmd_env(self):
         newenv['IMAGE'] = self.image or ""
 
         if hasattr(self.args, 'opt1') and self.args.opt1:
-            newenv['OPT1'] = self.args.opt1
+            newenv['OPT1'] = os.path.expandvars(self.args.opt1)
```
Comment (rhatdan) [inline Atomic/atomic.py]: The documentation defines these as UPPER CASE
```
man atomic-run
...
       OPT1, OPT2, OPT3
         Additional options which can be specified via the command.
```

Comment (baude) [inline Atomic/atomic.py]: OK LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by adelton** [comment]
> @adelton PTAL

Sorry for late reply. It looks good to me.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**